modules/src/Microsoft.Azure.IIoT.Modules.OpcUa.Publisher/src/Runtime/LegacyCliOptions.cs

```diff
@@ -272,45 +271,28 @@ public class LegacyCliOptions : Dictionary<string, string>, IAgentConfigProvider
         }
 
         private LegacyCliModel ToLegacyCliModel() {
-            return new LegacyCliModel {
-                Site = GetValueOrDefault(LegacyCliConfigKeys.PublisherSite, string.Empty),
```

Comment:
For `Site` which is now removed we still have the following option definition on line `58`:

```
                     { "s|site=", "The site OPC Publisher is working in.",
                        s => this[LegacyCliConfigKeys.PublisherSite] = s },
```

I think that it should be changed so something like the following, similar to how it is done for other legacy options on line `165`:

```
                     { "s|site=", "Legacy - do not use.", _ => {} },
```

There is also accompanying `LegacyCliConfigKeys.PublisherSite` field, that can be removed if we are not using `Site` any more.

Reply:
This should be done for the rest of the elements that have been removed as well.

Reply:
let's handle these in a separate PBI. "Site" seem to be used in module host. same with the others.